settings.gradle

```diff
@@ -11,8 +11,10 @@ includeWithFlags ':client:java-armeria',                    'java', 'publish', '
 includeWithFlags ':client:java-armeria-legacy',             'java', 'publish', 'relocate'
 includeWithFlags ':client:java-spring-boot-autoconfigure',  'java', 'publish', 'relocate'
 includeWithFlags ':client:java-spring-boot-starter',        'java', 'publish', 'relocate'
-includeWithFlags ':client:java-spring-boot1-autoconfigure', 'java', 'publish', 'relocate'
-includeWithFlags ':client:java-spring-boot1-starter',       'java', 'publish', 'relocate'
+includeWithFlags ':client:java-spring-boot1-autoconfigure', 'java', 'publish', 'relocate', 'no_aggregation'
+includeWithFlags ':client:java-spring-boot1-starter',       'java', 'publish', 'relocate', 'no_aggregation'
+includeWithFlags ':client:java-spring-boot3-autoconfigure', 'java17', 'publish', 'relocate', 'no_aggregation'
+includeWithFlags ':client:java-spring-boot3-starter',       'java17', 'publish', 'relocate', 'no_aggregation'
```

Comment:
It is difficult to infer if `java-spring-boot-autoconfigure` means Spring Boot 2 or 3 from the name. What do you think of renaming `java-spring-boot-{autoconfigure,starter}` to `java-spring-boot2-{autoconfigure,starter}` although they are breaking changes?

For consistency, `':testing:junit'` needs to be `':testing:junit5'.

Reply:
Done 😄

Reply:
- https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#new-autoconfiguration-annotation
`META-INF/spring.factories` should be migrated into `META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports`
  It is weird our tests weren't broken without the migration 🤔 
- What do you think of making `java-spring-boot3*` the main source set and referring them to `java-spring-boot2*`.

Reply:
> It is weird our tests weren't broken without the migration 🤔

`@SpringBootApplication` enables `@ComponentScan` on the same package by default.
I've removed this configuration and confirmed that tests fail without the `spring.factories` metadata.

> What do you think of making java-spring-boot3* the main source set and referring them in java-spring-boot2*.

I have mixed feelings about this. Although I think it can be clear which module has the source set, I also think this is more work for little benefit (both for the reviewer and code submitter)

Having said this, I guess major version-ups don't occur often so I've updated for this iteration.